extensions/ui/org.eclipse.smarthome.ui.paper/web/js/controllers.module.js

```diff
@@ -0,0 +1,105 @@
+capp.controller(
```

Comment:
You rely on a global variable *capp* here. This is defined at controllers.rules.js, which create a loading order dependency. Better way: `angular.module('PaperUI.controllers.rules').controller('addModuleDialogController', ...)` to get the module after all JS is loaded.